dev-util/gammaray/gammaray-2.9.0.ebuild

```diff
@@ -0,0 +1,90 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+QT_MINIMAL="5.9"
+
+inherit cmake-utils kde5-functions
+
+DESCRIPTION="High-level runtime introspection tool for Qt applications"
+HOMEPAGE="https://github.com/KDAB/GammaRay"
+
+if [[ ${PV} == 9999 ]] ; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/KDAB/GammaRay.git"
+else
+	SRC_URI="https://github.com/KDAB/GammaRay/releases/download/v${PV}/${P}.tar.gz"
+	KEYWORDS="~amd64"
+fi
+
+LICENSE="BSD"
+SLOT=0
+IUSE="auto-detect backtrace bluetooth +client\
+	concurrent designer -doc geolocation\
+	printsupport scxml svg test qt3d qml unwind\
+	wayland webengine"
+
+RDEPEND="
+	$(add_qt_dep qtcore)
+	$(add_qt_dep qtgui)
+	$(add_qt_dep qtnetwork)
+	$(add_qt_dep qtwidgets)
+	$(add_qt_dep qtxml)
+	$(add_frameworks_dep kitemmodels)
+	qt3d? ( $(add_qt_dep qt3d) )
```

Comment:
Qt team opted for `3d` indeed after consulting. If you don't change it back I can do it for you on merge, though. :)